### ISSUE TITLE
ESC was not responsed

### ISSUE THREAD

**Issue by chivstyle** [ismail-yilmaz/upp-components#7]
Hi.

I found another bug on windows.
```
U++: upp-win-15400
Toolchain: MSVS19x64
OS: Windows 10 64bits
```
Steps to reproduce the bug:
- connect to a remote linux machine
- open VIM
- press 'i' to insert anything
- press ESC, it can't exit from edit mode.

**Comment by ismail-yilmaz** [comment]
Thank you for the report.
Apparently the Windows key handling needs some more love and has more quirks than I initially assumed. (I'm usually on NIX variants...)
I was planning to refactor it anyway, so this provides me with a good excuse to fix it before I mark the 2020.2 tag.

I'll try to refactor the Windows key handling code on this weekend, so this will take several days.
Thank you for your patience.

**Comment by ismail-yilmaz** [comment]
@chivstyle ,

I have pushed the changes to improve key handling on Windows. (tested on Win 7/10)
Please check.

Key handling fix: 82c9b567c39355c6c79becbfdaaa0423feb2c41b
